docs/marketplace.service-management-area.md

```diff
@@ -0,0 +1,73 @@
+---
+id: marketplace.service-management-area
+title: Service Management Area
+---
+
+> Please note that the following functionality is only available to meshStacks that have the **Service Economy** module activated.
+> Reach out to your Customer Success representative if you are interested in using this module.
+
+## Introduction
+
+The Service Management Area can be used by workspaces to offer their [Platform Services](./marketplace.index.md#platform-services) to the rest of the organization.
+For example, the Azure Team in your organization could set up a workspace to offer their Azure [Platform](./administration.platforms.md) from there to
```

Comment:
Here its Azure Team upper case, in the next line Network team mixed case. I am in favor of having it just lower case